python-sdk/tutorials/automl-with-azureml/automl-nlp-ner/automl-nlp-ner.ipynb

```diff
@@ -287,6 +291,28 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also get the best run id and the best model with `get_output` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(\n",
+    "    best_run,\n",
+    "    best_model,\n",
+    ") = (\n",
+    "    automl_run.get_output()\n",
+    ")  # You might see a warning about \"enable_distributed_dnn_training\". Please simply ignore.\n",
```

Comment:
When will this warning be seen? In which case it will show up? Is there anything user needs to do to avoid this warning?

Reply:
I have checked with Adi. This warning shows because `enable_distributed_dnn_training` is not a permanent parameter for `AutoMLConfig`. It shows as long as we use distributed training. I have created a bug for you with some screenshots to have it fixed